lib/rom/repository/command_proxy.rb

```diff
@@ -11,11 +11,10 @@ class CommandProxy
 
       def initialize(command)
         @command = command
-        @root = Dry::Core::Inflector.singularize(command.name.relation).to_sym
       end
 
       def call(input)
-        command.call(root => input)
+        command.call(command.name.relation => input)
       end
```

Comment:
Reading that code comment made me feel a little better about changing this class. I think we're not far off making this obsolete. Could you point me in the direction of some cool bits of the command graph code to read?